src/pages/organize/[orgId]/journeys/[journeyId]/[instanceId]/index.tsx

```diff
@@ -9,46 +9,52 @@ import JourneyInstanceSidebar from 'components/organize/journeys/JourneyInstance
 import JourneyInstanceSummary from 'components/organize/journeys/JourneyInstanceSummary';
 import { organizationResource } from 'api/organizations';
 import { PageWithLayout } from 'types';
-import { scaffold } from 'utils/next';
 import SnackbarContext from 'hooks/SnackbarContext';
 import TimelineWrapper from 'components/TimelineWrapper';
+import { scaffold, ScaffoldedGetServerSideProps } from 'utils/next';
 import { ZetkinJourneyInstance, ZetkinPerson } from 'types/zetkin';
 
-const scaffoldOptions = {
+export const scaffoldOptions = {
   authLevelRequired: 2,
   localeScope: ['layout.organize', 'misc', 'pages.organizeJourneyInstance'],
 };
 
-export const getServerSideProps: GetServerSideProps = scaffold(async (ctx) => {
-  const { orgId, instanceId } = ctx.params!;
-
-  const { state: orgQueryState } = await organizationResource(
-    orgId as string
-  ).prefetch(ctx);
-
-  const { state: journeyInstanceQueryState } = await journeyInstanceResource(
-    orgId as string,
-    instanceId as string
-  ).prefetch(ctx);
-
-  if (
-    orgQueryState?.status === 'success' &&
-    journeyInstanceQueryState?.status === 'success'
-  ) {
-    return {
-      props: {
-        instanceId,
-        orgId,
-      },
-    };
-  } else {
-    return {
-      notFound: true,
-    };
-  }
-}, scaffoldOptions);
-
-interface JourneyDetailsPageProps {
+export const getJourneyInstanceScaffoldProps: ScaffoldedGetServerSideProps =
+  async (ctx) => {
+    const { orgId, instanceId } = ctx.params!;
+
+    const { state: orgQueryState } = await organizationResource(
+      orgId as string
+    ).prefetch(ctx);
+
+    const { state: journeyInstanceQueryState } = await journeyInstanceResource(
+      orgId as string,
+      instanceId as string
+    ).prefetch(ctx);
+
+    if (
+      orgQueryState?.status === 'success' &&
+      journeyInstanceQueryState?.status === 'success'
+    ) {
+      return {
+        props: {
+          instanceId,
+          orgId,
+        },
+      };
+    } else {
+      return {
+        notFound: true,
+      };
+    }
+  };
+
+export const getServerSideProps: GetServerSideProps = scaffold(
+  getJourneyInstanceScaffoldProps,
+  scaffoldOptions
+);
```

Comment:
Is there a reason to break this apart like this in this instance, rather than having the function inline as it is in other page modules?

Reply:
Ok now i have context for my reasoning! it was copied from how it's done in person page and its manage-tab, eg exporting them in the page so they can be recycled by the "other" tab (milestone or manage).

Reply:
Ah that makes sense! Ignore this then. 😊